src/components/scenes/ChangePasswordScene.tsx

```diff
@@ -45,28 +45,29 @@ const ChangePasswordSceneComponent = ({
   const [focusSecond, setFocusSecond] = React.useState(false)
   const [hidePassword, setHidePassword] = React.useState(true)
   const [spinning, setSpinning] = React.useState(false)
+  const [isShowError, setIsShowError] = React.useState(false)
 
   const hasPasswordStatus = useSelector(state => state.passwordStatus != null)
-  const passed = useSelector(state => state?.passwordStatus?.passed ?? false)
+  const isRequirementsMet = useSelector(
+    state => state?.passwordStatus?.passed ?? false
+  )
   const password = useSelector(state => state.create.password ?? '')
   const confirmPassword = useSelector(
     state => state.create.confirmPassword ?? ''
   )
-  const confirmError = useSelector(
+  const confirmPasswordErrorMessage = useSelector(
     state => state.create.confirmPasswordErrorMessage ?? ''
```

Comment:
Wow this is asinine.  `confirmPasswordErrorMessage` is a constant string but it is held in redux?! This whole scene needs a redux cleanup. For another day.